OpenEXR/IlmImf/ImfMisc.h

```diff
@@ -475,6 +475,13 @@ bool usesLongNames (const Header &header);
 IMF_EXPORT
 int getChunkOffsetTableSize(const Header& header,bool deprecated_attribute=false);
 
+
+//
+// return the number of scanlines in each chunk of a scanlineimage for the given scheme
+//
+IMF_EXPORT int
+numLinesInBuffer(Compression comp);
```

Comment:
Since this is declared in ImfCompressor.h, does it need to be declared here, too?